### ISSUE TITLE
missing client-id in Keys tab

### ISSUE THREAD

**Issue by KristijanL** [jaredhendrickson13/pfsense-api#461]
when i create an API key with latest v2 dev version i don't get client-id

**To Reproduce**
create API key

**Expected behavior**
Should show the client-id.

**Screenshots or Response**
![Screenshot 2024-05-09 at 19 34 35](https://github.com/jaredhendrickson13/pfsense-api/assets/977481/c89c802b-a376-4cf1-8f81-fe75d91d5090)

![Screenshot 2024-05-09 at 19 35 02](https://github.com/jaredhendrickson13/pfsense-api/assets/977481/46dc4d02-a155-4618-864d-d5d08c09d9bd)

**pfSense Version & Package Version:**
 - pfSense Version: 2.7.2-RELEASE
 - Package Version v2.0.0-dev-8e34317

**Affected Endpoints:**
 - all

**Comment by jaredhendrickson13** [comment]
Hey!

In v2 there is no concept of client-id or client-token like there was in v1. It is simply an API key that you include in your authorization header. For example: `Authorization: KeyAuth YOURKEYHERE`. The package knows which key is associated with which user already so there is no need for a client-id in v2.

Hope this helps.

Thanks!

**Comment by KristijanL** [comment]
i still have issues with the authentication via api token, with username/password works.

![Screenshot 2024-05-09 at 20 29 59](https://github.com/jaredhendrickson13/pfsense-api/assets/977481/05775599-f87f-4a8e-a329-25f7da524495)

```
Message from syslogd@fw at May  9 20:29:32 ...
php-fpm[94518]: /api/v2/status/carp/index.php: webConfigurator authentication error for user 'unknown' from: 192.168.5.21
```

**Comment by jaredhendrickson13** [comment]
Yeah I noticed that right after my previous response. I'll have a build later today with a fix for the malformed header. I'll mark this as a bug for now.

Thanks!

**Comment by jaredhendrickson13** [comment]
The OpenAPI spec doesn't really have a clean way to define the `Authorization` header with the `KeyAuth` identifier so I've opted to use the `x-api-key` header in its place. This has been addressed in the latest dev build and I've verified KeyAuth works as expected in the Swagger docs.
